src/javascript/app/App/Containers/SettingsDialog/settings-chart.jsx

```diff
@@ -70,8 +70,8 @@ const ChartSettings = ({
             <MediaItem>
             <MediaHeading>
                 <Localize
-                    str='Open-high-low-close [_1](OHLC) information[_2]'
-                    replacers={{ '1_2': <div /> }}
+                    i18n_default_text='Open-high-low-close <0>(OHLC) information</0>'
+                    components={[ <div key={0} /> ]}
```

Comment:
What is the purpose of `<0>`? Does it act as a replacer?

Reply:
@easteregg Yes. It's the index of the elements array passed to the `components` prop.